dependencies.toml

```diff
@@ -954,8 +947,14 @@ version.ref = "picocli"
 
 [libraries.prometheus]
 module = "io.prometheus:simpleclient_common"
+version.ref = "prometheus-legacy"
```

Comment:
Question) Is this used?

```suggestion
```

If it is used, should the `version.ref` be `prometheus` instead?

Reply:
Oops, forgot to remove it. Thanks!